corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -19,6 +19,13 @@
             plugins: {{ plugins|JSON }},
             features: {{ features|JSON }},
             core: {
+                dataSources: [
+                    {
+                        key: 'fixture',
+                        name: 'Fixtures',
```

Comment:
Check with @amsagoff on this name. I think we might want it to be "Lookup Tables" instead.